src/components/WebexMeetingGuestAuthentication/WebexMeetingGuestAuthentication.jsx

```diff
@@ -101,6 +101,7 @@ export default function WebexMeetingGuestAuthentication({
           ariaLabel={HINTS.name}
           autoFocus
           tabIndex={101}
+          withClear
```

Comment:
`withClear` should be a prop of InputField, and the TextInput component should have it hardcoded to `true`